### ISSUE TITLE
When printing Reflow sk, codec is ignored 

### ISSUE THREAD

**Issue by andrea-dintino** [dyne/Zenroom#436]
When printing the Reflow sk (only that), if a different encoding than default is chosen, the sk is printed as float (for any encoding, I tried them all) 

Code: 

```
# Here we are loading all the scenarios needed for the keys creations
Scenario 'ecdh': Create the key
Scenario 'ethereum': Create key
Scenario 'reflow': Create the key
Scenario 'schnorr': Create the key
Scenario 'eddsa': Create the key

Given nothing

# Here we are creating the keys
When I create the ecdh key
When I create the ethereum key 
When I create the reflow key
When I create the schnorr key
When I create the bitcoin key
When I create the eddsa key

Then print data as 'url64'

```
Output:

```
{
   "keyring": {
      "bitcoin": "CgGeJZbqfr5bUvQui2ZErwf0EIKvUzXuHMoWA9IsZmM",
      "ecdh": "_szWSLxKtoceTAuigHhZd32fHHQt1xmbD7Y7qeqhnFU",
      "eddsa": "Dx3I2Pv5fd1RfpPQ7iRJUZ6TldcPWNirl4pT4bJDHlw",
      "ethereum": "fn0DjqOodeCCWl2dNv1FReGgRlwLqyVvK5tQCaK-egQ",
      "reflow": "41594784144111293791036778249116432359897755545139326885090037583374363673608",
      "schnorr": "Lz4jIK-oI2o9GsgQitMCWGdIJx-sD3wEb9AipCIhb64"
   }
}

```

**Comment by jaromil** [comment]
Currently refactoring the way the keyring is handled in HEAP to improve security. Since SKs are sometimes encoded in their own peculiar way, there will be no possibility to print them in different encodings, but just as keyring.

**Comment by jaromil** [comment]
fixed by #438
